helm-chart/images/image-cleaner/Dockerfile

```
@@ -1,4 +1,5 @@
-FROM python:3.8-alpine3.11
+ARG DIST=buster
+FROM python:3.7-$DIST
```

Is there a reason for the variant and not the Python version to be an ARG? Maybe change the whole FROM:

```suggestion
FROM python:3.8-slim-buster
```

---

Changed